sdk/include/reactos/wine/test.h

```diff
@@ -668,14 +662,15 @@ const char *wine_dbgstr_wn( const WCHAR *str, intptr_t n )
         case '"':  *dst++ = '\\'; *dst++ = '"'; break;
         case '\\': *dst++ = '\\'; *dst++ = '\\'; break;
         default:
-            if (c >= ' ' && c <= 126)
-                *dst++ = (char)c;
-            else
+            if (c < ' ' || c >= 127)
             {
                 *dst++ = '\\';
-                sprintf(dst,"%04x",c);
-                dst+=4;
+                *dst++ = hex[(c >> 12) & 0x0f];
+                *dst++ = hex[(c >> 8) & 0x0f];
+                *dst++ = hex[(c >> 4) & 0x0f];
+                *dst++ = hex[c & 0x0f];
```

Comment:
This code is written for a 4-bytes WCHAR, which we don't have.
So the upper 2 digits will always be 0.

Reply:
Mark, Can you please review https://reactos.org/testman/diff.php?id1=56015182&id2=56024828&type=1&strip=0 and look at line #4279 on the left and the right and see if it makes sense to remove any of these lines? Thanks. Doug Lyons

Reply:
Sorry, I misread the code.
It is good as-is.